core/renderers/zelos/measurables/row_elements.js

```diff
@@ -10,30 +10,32 @@
  * @author samelh@google.com (Sam El-Husseini)
  */
 
-goog.provide('Blockly.zelos.RightConnectionShape');
+goog.module('Blockly.zelos.RightConnectionShape');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering.Measurable');
-goog.require('Blockly.blockRendering.Types');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.blockRendering.ConstantProvider');
+/* eslint-disable-next-line no-unused-vars */
+const ConstantProvider = goog.requireType('Blockly.blockRendering.ConstantProvider');
+const Measurable = goog.require('Blockly.blockRendering.Measurable');
+const Types = goog.require('Blockly.blockRendering.Types');
+const object = goog.require('Blockly.utils.object');
 
 
 /**
  * An object containing information about the space a right connection shape
  * takes up during rendering.
- * @param {!Blockly.blockRendering.ConstantProvider} constants The rendering
+ * @param {!ConstantProvider} constants The rendering
  *   constants provider.
  * @package
  * @constructor
- * @extends {Blockly.blockRendering.Measurable}
+ * @extends {Measurable}
  */
-Blockly.zelos.RightConnectionShape = function(constants) {
-  Blockly.zelos.RightConnectionShape.superClass_.constructor.call(this, constants);
-  this.type |= Blockly.blockRendering.Types.getType('RIGHT_CONNECTION');
+const RightConnectionShape = function(constants) {
+  RightConnectionShape.superClass_.constructor.call(this, constants);
+  this.type |= Types.getType('RIGHT_CONNECTION');
```

Comment:
I just had to read an article about bitwise operators in enums because of this O_o
Apparently I haven't poked the renderers very much before